src/style.scss

```diff
@@ -2,6 +2,9 @@
 @import './../node_modules/@alaskaairux/webcorestylesheets/dist/breakpoints';
 @import './../node_modules/@alaskaairux/webcorestylesheets/dist/core';
 
+/* Custom property to manipulate the height of auro-select element */
+$trigger-height: 40px;
```

Comment:
Recommend not using this. We are moving away from Sass variables as much as possible. I'd suggest creating local CSS variables. 

```CSS
:root {
  --trigger-height: 40px;
}
```

Ideally, we would want this to be governed by an Auro Design Token, but this will be addressed in a future update that is still in the works. https://github.com/AlaskaAirlines/AuroDesignTokens/issues/71

Reply:
Cannot create CSS variable in `:root` and use it in `:host`, therefore I created the variable in `:host`.